content/blog/pycon-india-tickets-2018.md

```diff
@@ -8,18 +8,20 @@ PyCon India 2018, the tenth edition of the premier conference in India on using
 
 The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. Although the Developer Sprints are free, we would be having registrations beforehand to manage the logistics. The ticket structure is as follows:- 
```

Comment:
extra space
```workshops,  keynotes``` -> ```workshops, keynotes,```

Reply:
@vipulgupta2048 Can you directly push a commit for this fix ?

Reply:
Should we hotfix these things or slip in any other issue as minor fix?

Reply:
Hotfixing, I just saw this. Thanks @realslimshanky